app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipesController < ApplicationController
+
+  def show
+    # require 'pry'; binding.pry
+    @recipe = Recipe.find(params[:id])
```

Comment:
This looks good! Will see if I can find the bug you were facing in another part of the application